app/views/partials/chargeSummary/ChargeSummaryNotCodingOutAndNotPayeSelfAssessment.scala.html

```diff
@@ -77,20 +83,22 @@
             <span>
             @{
                 if (interestCharge) chargeItem.latePaymentInterestAmount.getOrElse(throw MissingFieldException("Late Payment Interest Amount")).toCurrency
-                else chargeItem.originalAmount.toCurrency
+                else chargeItem.originalAmount.abs.toCurrency
```

Comment:
seem to me a change in logic, are you sure we need to use absolute value on this occasion? might be something to cross check with BAs